Dockerfile

```diff
@@ -17,13 +17,18 @@ LABEL org.opencontainers.image.documentation="https://hyp3-docs.asf.alaska.edu"
 # LABEL org.opencontainers.image.revision=""
 
 ARG DEBIAN_FRONTEND=noninteractive
+
 ENV PYTHONDONTWRITEBYTECODE=true
+ENV PROC_HOME=/home/conda/back-projection
+ENV MYHOME=/home/conda
 
-RUN apt-get update && apt-get install -y --no-install-recommends unzip vim && \
+RUN apt-get update && apt-get install -y --no-install-recommends unzip vim curl build-essential gfortran libfftw3-dev && \
     apt-get clean && rm -rf /var/lib/apt/lists/*
 
 ARG CONDA_UID=1000
 ARG CONDA_GID=1000
+ARG BACK_PROJECTION_TAG=0.2.0
```

Comment:
@forrestfwilliams @AndrewPlayer3 How do we want to manage the back_projection_tag as new versions of `hyp3-back-projection` code is released? Should that be a bumpless release after we merge to main? Or should we automate this to pull the latest projection (I haven't quite thought out the mechanics of updating this)?

Reply:
That version relates to the version of the [actual back-projection code](https://github.com/asfhyp3/back-projection). It's just a coincidence that the two repos have the same version right now! So I don't think we'll need to add another action.